app/src/main/res/values/strings.xml

```diff
@@ -952,4 +952,11 @@ Do you want to switch to this account?</string>
   <string name="you_can_return_to_complete_this_step_anytime_from_account_security_in_settings">You can return to complete this step anytime from Account Security in Settings.</string>
   <string name="confirm">Confirm</string>
   <string name="set_up_biometrics_or_choose_a_pin_code_to_quickly_access_your_vault_and_autofill_your_logins">Set up biometrics or choose a PIN code to quickly access your vault and AutoFill your logins.</string>
+  <string name="never_lose_access_to_your_vault">Never lose access to your vault</string>
+  <string name="the_best_way_to_make_sure_you_can_always_access_your_account">The best way to make sure you can always access your account is to set up safeguards from the start.</string>
+  <string name="create_a_hint">Create a hint</string>
+  <string name="your_hint_will_be_send_to_you_via_email_when_you_request_it">Your hint will be send to you via email when you request it.</string>
```

Comment:
```suggestion
  <string name="your_hint_will_be_send_to_you_via_email_when_you_request_it">Your hint will be sent to you via email when you request it.</string>
```

Reply:
There goes my side hustle as a stenographer! TY for catching.